source/blender/blenkernel/intern/sca.c

```diff
@@ -1188,6 +1191,23 @@ void BKE_sca_actuators_id_loop(ListBase *actlist, SCAActuatorIDFunc func, void *
   }
 }
 
+void BKE_python_components_id_loop(ListBase *complist, BKEPyComponentIDFunc func, void *userdata)
+{
+  PythonComponent *comp;
```

Comment:
This function and related included should go in source/blender/blenkernel/intern/python_component.c

Reply:
Fixed it. Thanks!